docs/DeveloperGuide.md

```diff
@@ -548,19 +548,30 @@ testers are expected to do more *exploratory* testing.
    1. Test case: `add i/S6516486H n/Bernard Lim p/98375489 e/b.lim@me.com a/80, Lorong 4 Toa Payoh, #12-34 c/7`<br>
       Expected: No new person is added. Error message is shown in the status bar.
 
+### Editing a person
+
+1. Editing a person's details
+   1. Prerequisites: List all people using the `list` command. There must be at least one person in the list.
+   
+   2. Test case: `edit 1 n/Jamus Lim p/97758933`
+      Expected: The person at index 1 is updated with the new name (Jamus Lim) and phone number (97758933). The updated details are shown in the list.
+   
+   3. Test case: `edit S1486256J a/Lorong 3 Toa Payoh, #12-34`
+      Expected: The person with NRIC S1486256J is updated with the new address (Lorong 3 Toa Payoh). The updated details are shown in the list.
+
```

Comment:
shld separate the test cases using NRIC and those using index
ie `Editing a person's details using INDEX`
and
`Editing a person's details using NRIC` or smth like this

and include a prerequisite of smth like
`Elderly with NRIC S1486256J exists or smth`